docs/source/_json/lock.resp

```diff
@@ -0,0 +1,12 @@
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "creator": "admin", 
+  "locked": true, 
+  "name": "plone.locking.stealable", 
+  "stealable": true, 
+  "time": 1477076400.0, 
```

Comment:
@buchi I am wondering if we have other "time" attributes in plone.restapi. We have to be sure, that we are consistent in what we expose.